lib/src/cartesian/hooks/useChartPressState.ts

```diff
@@ -15,7 +15,7 @@ export const useChartPressState = <Init extends ChartPressStateInit>(
   const state = React.useMemo(() => {
     return {
       isActive: makeMutable(false),
-      x: { value: makeMutable(0), position: makeMutable(0) },
+      x: { value: makeMutable(initialValues.x), position: makeMutable(0) },
```

Comment:
Nice catch here. We should probably add a `patch` changeset record for this so we can release this change.